Source/WebCore/html/ImageBitmapOptions.h

```diff
@@ -31,7 +31,7 @@
 namespace WebCore {
 
 struct ImageBitmapOptions {
-    enum class Orientation { None, FlipY };
+    enum class Orientation { None, FlipY, FromImage };
```

Comment:
This line: https://searchfox.org/wubkat/rev/33c49ff64449389431294ba0ace6f7d0ad6306b7/Source/WebCore/html/ImageBitmap.cpp#890 has a check for equality to None. Does this need to be adjusted if the behavior is the same as "none"?

Reply:
Oh, good catch! It does (but it just breaks an optimisation without the adjustment, not correctness),